lambda/rotator-snowflake/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"context"
+	"flag"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/chanzuckerberg/go-misc/lambda/rotator-snowflake/rotate"
+	"github.com/sirupsen/logrus"
+)
+
+var localFlag = flag.Bool("local", false, "Whether this lambda should be run locally")
+
+func Run(ctx context.Context) error {
+	return rotate.Rotate(ctx)
+}
+
+func main() {
+	logrus.SetLevel(logrus.DebugLevel)
+	flag.Parse()
+	logrus.Debugf("localFlag: %t", *localFlag)
+
+	// local-mode for lambda
+	if *localFlag {
+		err := Run(context.Background())
+		logrus.Fatal(err)
```

Comment:
huh, does this do the right thing if `err == nil`?

Reply:
Oh my gosh I should've checked for this

Reply:
It kind of errors even if err == nil. I wasn't sure how to handle it

Reply:
Since this is a pattern I've seen everywhere.

Reply:
if err != nil {
   logrus.Fatal(err)
}

would something like that work?

Reply:
oh yeah, that would totally work!